test/references/DataDryadAPI Drought.txt

```diff
@@ -0,0 +1,21 @@
+RegisterDataDep(
+    "Data from Drought intensification drives turnover of structure and function in stream invertebrate communities",
+    """
+	Dataset: Data from: Drought intensification drives turnover of structure and function in stream invertebrate communities
+	Website: https://datadryad.org/mn/object/http://dx.doi.org/10.5061/dryad.cc8834s
```

Comment:
This URL is wrong
The URL you should be linking to is
http://dx.doi.org/10.5061/dryad.cc8834s